### DISCUSSION TITLE
cluster networks contains all pods check fails

### DISCUSSION THREAD

**Discussion by ranjithsaccount** [linkerd/linkerd2#9336] [Help] [answered]
I am new to service meshes and getting below error why trying to inject linkerd to particular namespace:

**cluster networks contains all pods
    the Linkerd clusterNetworks ["<List of IPADDRESS"] do not include pod default/<podname> (<ip>)**

Could you please suggest how to fix this or there is a workaround to update linkerd cluster networks. Appreciate any help

**Comment by wmorgan** [comment, marked as answer]
This was answered in Slack. Add your Pod CIDR to Linkerd's `clusterNetworks` setting will fix this.

**Reply by Vinaum8** [reply]
@wmorgan I opened slack and couldn't find the answer, could you post the slack link here?

**Reply by wmorgan** [reply]
The magic of Slack is that the answers disappear after 90 days. But you should be able to follow the directions here: https://linkerd.io/2.15/tasks/troubleshooting/#l5d-cluster-networks-pods

**Reply by Vinaum8** [reply]
Well, i edited configmap from linkerd with my vpc cidr and worked.
